\hypertarget{_test_exceptions_8hpp}{\section{/home/david/\-Dropbox/\-Development/\-Chaos\-Core/\-Chaos\-Core/src/cxx/chaoscore/test/\-Test\-Exceptions.hpp File Reference}
\label{_test_exceptions_8hpp}\index{/home/david/\-Dropbox/\-Development/\-Chaos\-Core/\-Chaos\-Core/src/cxx/chaoscore/test/\-Test\-Exceptions.\-hpp@{/home/david/\-Dropbox/\-Development/\-Chaos\-Core/\-Chaos\-Core/src/cxx/chaoscore/test/\-Test\-Exceptions.\-hpp}}
}
{\ttfamily \#include \char`\"{}chaoscore/base/\-Base\-Exceptions.\-hpp\char`\"{}}\\*
\subsection*{Classes}
\begin{DoxyCompactItemize}
\item 
class \hyperlink{classchaos_1_1test_1_1ex_1_1_test_error}{chaos\-::test\-::ex\-::\-Test\-Error}
\begin{DoxyCompactList}\small\item\em Warns of an unexpected error during testing procedures. \end{DoxyCompactList}\item 
class \hyperlink{classchaos_1_1test_1_1ex_1_1_test_decleration_error}{chaos\-::test\-::ex\-::\-Test\-Decleration\-Error}
\begin{DoxyCompactList}\small\item\em Warns that an error has occurred during test deceleration time. \end{DoxyCompactList}\item 
class \hyperlink{classchaos_1_1test_1_1ex_1_1_test_runtime_error}{chaos\-::test\-::ex\-::\-Test\-Runtime\-Error}
\begin{DoxyCompactList}\small\item\em Warns that an error has occurred during test runtime. \end{DoxyCompactList}\end{DoxyCompactItemize}
\subsection*{Namespaces}
\begin{DoxyCompactItemize}
\item 
\hyperlink{namespacechaos}{chaos}
\begin{DoxyCompactList}\small\item\em the global Chaos\-Core namespace which contains everything within Chaos\-Core. \end{DoxyCompactList}\item 
\hyperlink{namespacechaos_1_1test}{chaos\-::test}
\begin{DoxyCompactList}\small\item\em Chaos\-Core's testing module. \end{DoxyCompactList}\item 
\hyperlink{namespacechaos_1_1test_1_1ex}{chaos\-::test\-::ex}
\begin{DoxyCompactList}\small\item\em Exceptions relating to Chaos\-Core's testing module. \end{DoxyCompactList}\end{DoxyCompactItemize}


\subsection{Detailed Description}
\begin{DoxyAuthor}{Author}
David Saxon 
\end{DoxyAuthor}
